Lib/test/test_sys.py

```diff
@@ -11,6 +9,9 @@
 import sys
 import sysconfig
 import test.support
+from test import support
+from test.support.script_helper import assert_python_ok, assert_python_failure
+from test import threading_helper
```

Comment:
wait, it lives into test.support, not test, no?

Reply:
good catch, thanks.